### PR TITLE
Fix clang build

### PR DESCRIPTION
**Pull request by kgaillot**
... with a couple of other small changes

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this short PR? Thanks

File: lib/pacemaker/pcmk_sched_actions.c [hunk 1]
```diff
@@ -631,7 +631,7 @@ pcmk__update_action_for_orderings(pe_action_t *then, pe_working_set_t *data_set)
                          "Disabled ordering %s then %s in favor of %s then %s",
                          other->action->uuid, then->uuid, first->uuid,
                          then->uuid);
-            other->type = pcmk__ar_none;
+            other->type = (enum pe_ordering) pcmk__ar_none;
         }
```
Comment (clumens) [inline lib/pacemaker/pcmk_sched_actions.c:635]: clang is also complaining about the use at lib/pacemaker/pcmk_sched_ordering.c:1115 for me.

Comment (kgaillot) [reply lib/pacemaker/pcmk_sched_actions.c:635]: whoops, thanks

**Comment by kgaillot** [comment]
updated per review

**Review by clumens — APPROVED** [review]

**Pull request merged**